src/main/java/org/broadinstitute/hellbender/utils/pileup/UnifiedPileupElementTracker.java

```diff
@@ -0,0 +1,94 @@
+package org.broadinstitute.hellbender.utils.pileup;
+
+import htsjdk.samtools.SAMFileHeader;
+import org.broadinstitute.hellbender.utils.read.ReadUtils;
+
+import java.util.Collections;
+import java.util.Iterator;
+import java.util.List;
+import java.util.Objects;
+import java.util.Set;
+import java.util.Spliterator;
+import java.util.Spliterators;
+import java.util.function.Predicate;
+import java.util.stream.Collectors;
+import java.util.stream.Stream;
+
+/**
```

Comment:
Ditto about docs, because I don't understand why the SingleSamplePileupElementTracker inherits from Unified and PerSample inherits from PileupElementTracker.  If you could just address when a developer would instantiate each.

Reply:
Added a note for the inheritance and when they are supposed to be used.